### PR TITLE
Adding tests to wrangler-refactor

### PR DESCRIPTION
**Pull request by isichei**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,98 @@
...
+pydb.read_sql("SELECT * from a_database.table LIMIT 10")
+
+# Create a temp table to do further seperate SQL queries later on
+pydb.create_temp_table("SELECT a_col, count(*) as n FROM a_database.table GROUP BY a_col", table_name="temp_table_1")
+df = pydb.read_sql_query("SELECT * FROM __temp__.temp_table_1 WHERE n < 10")
+```
+
+### More advanced usage
+
+Get the actual name for your temp database, create your temp db then delete it using awswrangler (not awsrangler will raise an error if the database does not exist)
```
Comment (gkelly900) [inline README.md:71]: (~~not~~)

File: README.md [hunk 2]
```diff
@@ -1,5 +1,98 @@
...
+> Requires a pip release above 20.
+
+```bash
+pip install "pydbtools @ git+https://github.com/moj-analytical-services/pydbtools"
+
+## Or install a specific release
+
+pip install "pydbtools @ git+https://github.com/moj-analytical-services/pydbtools@v3.0.0"
+```
+
```
Comment (gkelly900) [inline README.md:14]: I think we should add a quickstart section first. e.g.:

## Quickstart guide

### Read an SQL Athena query into a pandas dataframe

```python
import pydbtools as pydb
df = pydb.read_sql("SELECT * from a_database.table LIMIT 10")
```
### Run a query in Athena using pydbtools

```python
response = pydb.start_query_execution_and_wait("CREATE DATABASE IF NOT EXISTS my_test_database")
```


### Create a temporary table to do further separate SQL queries on later

```python
pydb.create_temp_table("SELECT a_col, count(*) as n FROM a_database.table GROUP BY a_col", table_name="temp_table_1")
df = pydb.read_sql_query("SELECT * FROM __temp__.temp_table_1 WHERE n < 10")
```

**Review by gkelly900 — COMMENTED** [review]
Looks good! I have made some suggested changes, lemme know if they don't make sense.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,98 @@
 # pydbtools
 
+## Installation
+
+> Requires a pip release above 20.
+
+```bash
+pip install "pydbtools @ git+https://github.com/moj-analytical-services/pydbtools"
```
[the remark this answers is not in the document]
Comment (isichei) [reply README.md:8]: Not sure --- probably should but less bothered about releasing on pypi now that pip installs smoothly from github.

[the remark this answers is not in the document]
Comment (gkelly900) [reply README.md:8]: I reckon users will find it helpful

**Review by gkelly900 — APPROVED** [review]

**Pull request merged**